### PR TITLE
Optionally use the center of the beacon's place to populate its latitude & longitude fields.

### PR DESCRIPTION
**Pull request by peetsasaki**

### CODE REVIEW

File: samples/python/pbapi/__init__.py [hunk 1]
```diff
@@ -303,6 +303,12 @@ def register_beacon(self, arguments):
                                  action='store_false', dest='ibeacon_props',
                                  help='If beacon.advertisedId.type is IBEACON, parse out the UUID, major, minor IDs ' +
                                       'and create matching ibeacon_ properties for each.')
+        args_parser.add_argument('--maps-api-key', metavar='API_KEY',
+                                 help='Maps API key with which to call geocoder or places APIs. Must at minimum have ' +
+                                      'the geocoder API active.')
+        args_parser.add_argument('--set-latlng-from-place',
+                                 action='store_true',
+                                 help='Use the center of the place as the latitude and longitude of the beacon.')
```
Comment (stefano-maggiolo) [inline samples/python/pbapi/__init__.py]: I think you should clarify what happens if both sources are present and the flag is true

File: samples/python/pbapi/__init__.py [hunk 2]
```diff
@@ -802,6 +833,47 @@ def set_property(self, arguments):
...
+
+        if 'placeId' not in beacon:
+            if DEBUG:
+                print('No place_id in beacon, cannot infer its lat/lng.')
+            return
+
+        if DEBUG:
+            print('Attempting to set lat/lng based on place id {}'.format(beacon['placeId']))
+
+        places_api_url = 'https://maps.googleapis.com/maps/api/place/details/json?placeid={}&key={}'.format(beacon['placeId'], maps_api_key)
```
Comment (stefano-maggiolo) [inline samples/python/pbapi/__init__.py]: Wrap

File: samples/python/pbapi/__init__.py (hunk 1 above)
Comment (peetsasaki) [reply samples/python/pbapi/__init__.py]: Done.

File: samples/python/pbapi/__init__.py (hunk 2 above)
Comment (peetsasaki) [reply samples/python/pbapi/__init__.py]: Done.

**Pull request merged**